content/developers/api-reference/events-api/index.md

```diff
@@ -91,6 +91,75 @@ Additional YAML examples can be found in the articles in the [Overview](/platfor
   }
   ```
 
+### DataTrails Reserved Attributes
+
+The DataTrails platform as reserved attributes starting with `arc_`.
+Asset-Events have the following reserved attributes:
+
+- **`arc_display_type`**: The value used to display in the DataTrails Application.
```

Comment:
Move the content to be a pointer to: https://docs.datatrails.ai/glossary/reserved-attributes/